drivers/bridge/bridge.go

```diff
@@ -735,7 +735,11 @@ func (d *driver) createNetwork(config *networkConfiguration) error {
 
 	// Apply the prepared list of steps, and abort at the first error.
 	bridgeSetup.queueStep(setupDeviceUp)
-	return bridgeSetup.apply()
+	if err = bridgeSetup.apply(); err != nil {
```

Comment:
would be better if you change the method prototype like this:
```
func (d *driver) createNetwork(config *networkConfiguration) (err error) {
```
and remove line 618
that way we are sure that if the body of this function has a return the defer is still being honored

Reply:
If we were going that route I'd probably suggest something less generic than `err`.   It's almost idiomatic to shadow it.